cli/src/commands/git.rs

```diff
@@ -161,6 +161,17 @@ pub struct GitPushArgs {
     /// https://martinvonz.github.io/jj/latest/revsets#string-patterns.
     #[arg(long, short, value_parser = parse_string_pattern)]
     branch: Vec<StringPattern>,
+    /// Push all tracked branches (including deleted branches)
+    ///
+    /// This usually means that the branch was already pushed to or fetched from
+    /// the relevant remote. For details, see
+    /// https://martinvonz.github.io/jj/latest/branches#remotes-and-tracked-branches
```

Comment:
FYI: This heading does not exist on the stable version docs yet, but should appear there on the next release, which should happen this week.